python/xorq/common/utils/dask_normalize/dask_normalize_expr.py

```diff
@@ -535,7 +536,11 @@ def normalize_op(op, compiler=None):
         compiler=compiler,
     )
     reads = op.find(rel.Read)
-    dts = op.find((ir.DatabaseTable, rel.FlightExpr, rel.FlightUDXF))
+    dts = tuple(
+        node
+        for node in op.find((ir.DatabaseTable, rel.FlightExpr, rel.FlightUDXF))
```

Comment:
Both FlightExpr and FlightUDXF inherit from DatabaseTable; should I remove them from the tuple? @dlovell